app/upgrades/v16/upgrades_test.go

```diff
@@ -0,0 +1,75 @@
+package v16_test
+
+import (
+	"strings"
+	"testing"
+
+	ratelimittypes "github.com/Stride-Labs/ibc-rate-limiting/ratelimit/types"
+	"github.com/stretchr/testify/require"
+
+	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
+
+	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	channeltypes "github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
+
+	sdkmath "cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+
+	"github.com/cosmos/gaia/v16/app/helpers"
+	v16 "github.com/cosmos/gaia/v16/app/upgrades/v16"
+)
+
+var AtomSupply = sdkmath.NewInt(1000)
+
+func TestAddRateLimits(t *testing.T) {
+	gaiaApp := helpers.Setup(t)
+	ctx := gaiaApp.NewUncachedContext(true, tmproto.Header{})
+	ratelimitkeeper := gaiaApp.RatelimitKeeper
+
+	// mint atoms
+	amount := sdk.NewCoin(v16.RateLimitDenom, AtomSupply)
+	amountCoins := sdk.NewCoins(amount)
+	err := gaiaApp.BankKeeper.MintCoins(ctx, minttypes.ModuleName, amountCoins)
+	require.NoError(t, err)
+
+	// mock IBC channels
+	gaiaApp.IBCKeeper.ChannelKeeper.SetChannel(ctx, transfertypes.PortID, v16.RateLimits["osmosis-1"].ChannelId, channeltypes.Channel{})
```

Comment:
Similarly as above, why not have a `for` loop for this?

Reply:
Same answer as above.